compiler/AST/build.cpp

```diff
@@ -610,13 +626,15 @@ BlockStmt* buildImportStmt(Expr* mod, std::vector<PotentialRename*>* names,
     }
   }
 
-  ImportStmt* newImport = new ImportStmt(mod, privateImport, &namesList,
+  // Set the privacy to a dummy value until we know what it should be (which
+  // happens when we are done determining how many subexpressions there are)
+  ImportStmt* newImport = new ImportStmt(mod, true, &namesList,
```

Comment:
This appears several times here. So wouldn't it be better to remove the privacy from the ImportStmt constructor's arguments?

Reply:
I thought about that, but there's other situations where we create import statements in the compiler and those situations know the intended privacy.  I decided not to duplicate constructors, but I suppose I could make it an argument with a default value and use that . . .

Reply:
In that event, I'd consider making the other code construct the ImportStmt and then set the privacy. But what you have here is also OK if you decide that's the better way for now.